tests/pools/test_pool_rpc.py

```diff
@@ -843,8 +934,6 @@ async def test_leave_pool(self, setup, trusted_and_fee, self_hostname):
             PeerInfo(self_hostname, uint16(full_node_api.full_node.server._port)), None
         )
 
-        WAIT_SECS = 200
```

Comment:
I see, we used a constant like this already. I think we should still leave it at 10 seconds though